app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Admin
+  class UsersController < ApplicationController
+
+    def index
+      @users = User.all
+    end
+
+    def destroy
+
+      User.find(params[:id]).destroy
```

Comment:
it's important to authorize the user as an admin for them to destroy the user. A malicious user that does not have the admin role could arbitrarily delete users without such protection.